app/javascript/src/components/Dashboard/Contacts/DeleteAlert.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+import { Alert, Toastr } from "neetoui";
+
+const DeleteAlert = ({ onClose }) => (
+  <Alert
+    isOpen
+    message="Are you sure you want to continue? This cannot be undone."
```

Comment:
`continue` with what? Not sure why I have this out again and again. Please go through the Slack thread.

Reply:
Sorry I missed this part. I have corrected my mistake and updated the code.